routes/user_get.go

```diff
@@ -0,0 +1,18 @@
+package routes
+
+import (
+        "github.com/fatih/color"
+	"github.com/gominima/minima"
+)
+
+func UserGet() *minima.Router {
+	rt := minima.NewRouter()
+        rt.Post("/:id", func(res *minima.Response, req *minima.Request) {
+		//This is a super simple create method without any logic implemented!
+		res.Status(200)
+		param := req.GetParam("id")
+		color.Cyan("User id is %v", param)
```

Comment:
@apoorvcodes Indentations are both spaces and tabs? Wouldn't that mess up linters and stuff?

![image](https://user-images.githubusercontent.com/38729705/152159940-8c3893d2-0763-4417-a3ec-ba08d010cd1d.png)

Reply:
I ran a lint works good

Reply:
Alright then. You might fix the incorrect indentation and stuff.